packages/db/drizzle/0005_careful_union_jack.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE "account" ALTER COLUMN "email" SET DEFAULT '';
```

Comment:
You need to recreate these migrations (preferably into a single one). 

They currently fail when I try apply to a database with the schema that existed before this PR.